src/main/java/org/mitre/hapifhir/BackendAuthorizationInterceptor.java

```diff
@@ -46,6 +46,9 @@ public BackendAuthorizationInterceptor(String authServerCertsAddress) {
   
   @Override
   public List<IAuthRule> buildRuleList(RequestDetails theRequestDetails) {
+    String requireAuth = System.getenv("REQUIRE_AUTH");
+    if (requireAuth == null || requireAuth.equals("false")) return authorizedRule();
```

Comment:
I feel like the default (i.e. `REQUIRE_AUTH` unset) should be `true`. I know this isn't a production environment but I always feel like security should be on by default unless explicitly said otherwise.